ratify.yaml

```
@@ -0,0 +1,157 @@
+package:
+  name: ratify
+  version: "1.4.0"
+  epoch: 0
+  description: Artifact Ratification Framework (CNCF Sandbox)
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: f3f6229dee1051e4beb580ec17ad68b7a45f23af
+      repository: https://github.com/notaryproject/ratify
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.38.0
+        github.com/go-jose/go-jose/v3@v3.0.4
+        github.com/go-jose/go-jose/v4@v4.0.5
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        github.com/golang-jwt/jwt/v5@v5.2.2
+        github.com/cloudflare/circl@v1.6.1
+        github.com/open-policy-agent/opa@v1.4.0
+        github.com/go-viper/mapstructure/v2@v2.3.0
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -X github.com/ratify-project/ratify/internal/version.GitCommitHash=$(git rev-parse HEAD)
+        -X github.com/ratify-project/ratify/internal/version.GitTag=${{package.version}}
+      output: ratify
+      packages: ./cmd/ratify
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/opt/ratify
+      install -Dm644 config/config.json ${{targets.contextdir}}/opt/ratify/config.json
+
+data:
+  - name: plugins
+    items:
+      sbom:
+      licensechecker:
+      schemavalidator:
+      vulnerabilityreport:
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package for the ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/app
+          ln -sf /usr/bin/ratify ${{targets.contextdir}}/app/ratify
+    test:
+      pipeline:
+        - runs: test "$(readlink /app/ratify)" = "/usr/bin/ratify"
+
+  - range: plugins
+    name: ${{package.name}}-${{range.key}}
+    description: Ratify ${{range.key}} plugin
+    pipeline:
+      - uses: go/build
+        with:
+          output: ${{range.key}}
+          packages: ./plugins/verifier/${{range.key}}
+
+  - range: plugins
+    name: ${{package.name}}-${{range.key}}-compat
+    description: Ratify ${{range.key}} compatibility package
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/ratify/plugins
+          ln -sf /usr/bin/${{range.key}} ${{targets.contextdir}}/opt/ratify/plugins/${{range.key}}
+    test:
+      pipeline:
+        - runs: test "$(readlink /opt/ratify/plugins/${{range.key}})" = "/usr/bin/${{range.key}}"
+
+update:
+  enabled: true
+  github:
+    identifier: notaryproject/ratify
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+  pipeline:
+    - runs: |
+        curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notation.crt
```

```suggestion
        curl -sSLO https://raw.githubusercontent.com/notaryproject/ratify/cb7ffd11a99446731468de435b43f44d4afeaa49/test/testdata/notation.crt
```
If we absolutely cant use a self-signed certificate we should at least pin this to a specific commit just to make sure this wont blow up due to repository restructuring